src/sentry/api/endpoints/organization_member/__init__.py

```diff
@@ -105,8 +105,6 @@ def get_allowed_org_roles(
 
     if is_active_superuser(request):
         return roles.get_all()
-    if not request.access.has_scope("member:admin"):
```

Comment:
I think this should still return but with `request.access.has_scope(member:admin) or (member:invite)`. no?

Reply:
We use this function in places where we do want to gate it to `member:admin` only, so unfortunately the only workaround is a new arg

Reply:
@ameliahsu can you please add a comment here? I think it can be confusing for future readers.